docs/en/learning_content/toolkit_basics_guides/advanced_config.md

```diff
@@ -0,0 +1,241 @@
+4. A configuration folder named "the_other_side.” Inside the "configs” folder where you store your configs, create a folder using this project name. We will be creating a project called "the_other_side" in this exercise.
+ 
+  
+## About the Advanced Project Setup Wizard
+
+The Advanced Project Setup Wizard in Shotgun Desktop generates a Default Configuration that provides a solid base to build on adding key software integrations, customizable settings, apps and UI elements that support the pipeline process. It creates a configuration you can edit and extend to met your project’s pipeline needs.
+
+The Default Configuration includes: 
+* A basic filesystem schema and templates that determine where files live on disk 
+* All of the supported [software integrations](https://support.shotgunsoftware.com/hc/en-us/articles/219039798-Integrations-Apps-and-Engines) allowing file management from directly inside the user’s software applications. 
+
+Customizations are only limited by imagination, cleverness, and programming knowledge or the ability to borrow from what others in the Shotgun community have created.
+ 
+## Extending the Default Configuration 
+
+A configuration is required for every project. The first time a project is accessed through Shotgun Desktop a Basic Configuration is generated by Shotgun Toolkit. This Basic Configuration automatically detects the supported software applications a user has on their system. Settings in the pipeline configuration govern the integrations within the supported software applications that display project related information and the apps that manage project files. The project information is displayed through the integrated display [Panel]( https://support.shotgunsoftware.com/hc/en-us/articles/219033098-Shotgun-Panel). Publishes are managed by publishing files with the [Publisher]( https://support.shotgunsoftware.com/hc/en-us/articles/219032998-Publishing-your-work). Uploading files is managed through the [Loader]( https://support.shotgunsoftware.com/hc/en-us/articles/219033078-Load-Published-Files-). The Basic Configuration does not include filesystem management setup or the development of templates for specifying how files and directories are named on your disk. It also doesn’t have the plethora of Apps and hooks that are added when a Default Configuration is created. It is a bare bones configuration that can be manually edited to be whatever you would like it to be. The Wizard automatically generates a Default Configuration based on the settings you choose and has more Apps, hooks and reference guides to support you on your journey. This guide will expand your mind to the possibilities. 
+
+## Difference between the Default Configuration and Basic Configuration.
+
+## SHOTGUN CONFIGURATIONs
+
+| BASIC CONFIGURATION | DEFAULT CONFIGURATION |
```

Comment:
Some feedback on the table:
![image](https://user-images.githubusercontent.com/337710/52484586-9e33a780-2bae-11e9-880f-c01a8569e3ad.png)

- totally fine that the two columns aren't same width. Don't worry about it.
- Please make the text left aligned. It looks a little strange right now.
- basic software integrations and all the bullet points can be in a single table cell, don't need separate cells for each bullet point.
- I THINK BETTER TO AVOID CAPITALIZATION - LOOKS A LITTLE SHOUTY! Instead use *bold* or _italic_ to emphasize.
- Rather than "The Panel", say "Shotgun Panel"

Reply:
Wonder if we could have three columns in this table? So start with a column to the left with some 'headings' - Right now it almost feels like that's something that's gone missing. What is the description of each row?